clean_files.txt

```diff
@@ -50,15 +50,16 @@ themes/purity
 
 # plugins
 #
+plugins/available/alias-completion.plugin.bash
 plugins/available/basher.plugin.bash
 plugins/available/cmd-returned-notify.plugin.bash
 plugins/available/docker-machine.plugin.bash
 plugins/available/git.plugin.bash
 plugins/available/go.plugin.bash
 plugins/available/goenv.plugin.bash
-plugins/available/history.plugin.bash
 plugins/available/history-search.plugin.bash
 plugins/available/history-substring-search.plugin.bash
+plugins/available/history.plugin.bash
```

Comment:
Logically, this change feels wrong, but technically, `-` _does_ sort higher than `.` - I vote that this change is good because we'll probably move to some automated sorting concept for this file ( See #1871 ) and it too will likely result in this sorting order.